docs/GitBranchSetup.md

```diff
@@ -32,21 +32,21 @@ primary **accessibility-insights-windows** repository.
 -   When you run git remote -v, you should see only your fork in the output list
 
 ```
-> git remote -v
+git remote -v
 origin  https://github.com/ada-cat/accessibility-insights-windows (fetch)
 origin  https://github.com/ada-cat/accessibility-insights-windows (push)
 ```
 
 -   Map the primary **accessibility-insights-windows** as the upstream remote
 
 ```
-> git remote add upstream https://github.com/Microsoft/accessibility-insights-windows
+git remote add upstream https://github.com/Microsoft/accessibility-insights-windows
 ```
 
 -   Now, running `git remote -v` should show the upstream repository also
 
 ```
-> git remote -v
+git remote -v
 origin  https://github.com/ada-cat/accessibility-insights-windows (fetch)
```

Comment:
I could understand if someone feels strongly that the command here should be distinct from the output, suggestions are welcome